docs/src/main/paradox/elasticsearch.md

```diff
@@ -98,6 +98,7 @@ Java
 | ---------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------ |
 | bufferSize             | 10      | `ElasticsearchSource` retrieves messages from Elasticsearch by scroll scan. This buffer size is used as the scroll size. | 
 | includeDocumentVersion | false   | Tell Elasticsearch to return the documents `_version` property with the search results. See [Version](http://nocf-www.elastic.co/guide/en/elasticsearch/reference/current/search-request-version.html) and [Optimistic Concurrenct Control](https://www.elastic.co/guide/en/elasticsearch/guide/current/optimistic-concurrency-control.html) to know about this property. |
+| scrollDuration         | FiniteDuration(5, TimeUnit.MINUTES)      | `ElasticsearchSource`  retrieves messages from Elasticsearch by scroll scan. This parameter is used as a scroll value and will be converted to a string value in the following format: s"$Length$Unit". ie: 5m, 10ms. See [Time units](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#time-units) for supported units.                |
```

Comment:
Just stating `5 min` as default is enough. 
You don't need to explain how the conversion is done.